### ISSUE TITLE
Feedback 💕 

### ISSUE THREAD

**Issue by transitive-bullshit** [tobiasbueschel/search-gpt#6]
This project looks awesome!

Some misc thoughts / questions:

1. Maybe consider adding support for https://serpapi.com
2. I really love the logo / icon. Curious what your process was for making it?
3. If you're not already in it, you should consider joining our [ChatGPT Hacker's Discord](https://discord.gg/v9gERj825w) 👋 Most of the chatgpt-related OSS maintainers from the JS/python communities are active on there

Cheers :)

**Comment by tobiasbueschel** [comment]
Hi @transitive-bullshit thanks for the suggestions and great to hear that you like the project! To reply to your thoughts:

1. It's just a PoC, but for now I'm thinking to mainly support official APIs i.e., the direct APIs from Google, Twitter etc.
2. Thanks, I've used Midjourney for it, idea was that the stem of the tree symbolizes the LLM and the different branches are the sources (i.e. APIs) it pulls from. I think it could be improved though.
3. Thanks, would be happy to!

Btw, I noticed that you've built https://github.com/transitive-bullshit/chatgpt-api#usage---chatgptapi maybe I should integrate the package to e.g., leverage streaming and built this into the CLI :)
